README.md

```diff
@@ -92,381 +92,172 @@
 <br clear="all" />
 
 ------------------------------------------------------------------------
+# Highlights
 
-Ivy is both an ML transpiler and a framework, currently supporting JAX,
-TensorFlow, PyTorch, and Numpy.
+Ivy is an open-source machine learning framework that
+enables you to:
 
-Ivy unifies all ML frameworks 💥 enabling you not only to **write code
-that can be used with any of these frameworks as the backend**, but also
-to **convert 🔄 any function, model, or library written in any of them to
-your preferred framework!**
+- 🔥 **Autotune your model**: for optimal framework, compiler, and hardware selection.
+- 💥 **Boost model speed**:  by up to 3x.
+- 🔄 **Convert code code into any framework**: to use and build of top of any model, library, infrastructure or device by converting any code between frameworks.
+- ⚒️ **Write framework-agnostic code**: by letting you choose your preferred ML framework as a backend. to leverage all its benefits and tools
```

Comment:
Also I think this can be more informative while keeping it short, e.g.

---
- 🔥 **Autotune your model**: Automatically find the optimal framework, compiler infrastructure and hardware for your specific use case using `ivy.autotune`.
- 💥 **Boost model speed**:  Speed up your models by up to 3x using the most efficient ML infrastructure with `ivy.compile`.
- 🔄 **Convert code into any framework**: Use and build on top of any model, library, or device by converting any code from one framework to another using `ivy.transpile`.
- ⚒️ **Write framework-agnostic code**: Write your code once in ivy and then choose the most appropriate ML framework as the backend to leverage all the benefits and tools.
---

Reply:
I feel this should be like a hook rather than too informative, even if we are talking to engineer, getting them curious with short time was the main, I changed it to your suggestions though.

Reply:
Thanks! Yeah I get your point, I don't really think the current ones are too informative tho

Reply:
Sure, thanks for that! I will count this as resolved.